app-modules/team/src/Filament/Resources/TeamResource.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Assist\Team\Filament\Resources;
+
+use Assist\Team\Models\Team;
+use Filament\Resources\Resource;
+use Assist\Team\Filament\Resources\TeamResource\Pages\EditTeam;
+use Assist\Team\Filament\Resources\TeamResource\Pages\ListTeams;
+use Assist\Team\Filament\Resources\TeamResource\Pages\CreateTeam;
+use Assist\Team\Filament\Resources\TeamResource\RelationManagers\UsersRelationManager;
+
+class TeamResource extends Resource
+{
+    protected static ?string $model = Team::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    public static function getRelations(): array
+    {
+        return [
+            UsersRelationManager::make(),
+        ];
+    }
+
+    public static function getPages(): array
+    {
+        return [
+            'index' => ListTeams::route('/'),
+            'create' => CreateTeam::route('/create'),
+            'edit' => EditTeam::route('/{record}/edit'),
```

Comment:
Let's go ahead and add a View page as well. There will most likely need to be Users that can view a team and see info about it, but not edit.